src/NBInclude.jl

```diff
@@ -56,10 +56,20 @@ If the cell has no number (e.g. if it hasn't been evaluated yet), then
 it is assigned a number `+N` for the `N`-th nonempty cell.  If `renumber`
 is set to `true`, then the cell numbers saved in the notebook are ignored
 and each cell is assigned a consecutive number `N`.
+
+`counters` and `regex` can be used to limit the execution of notebook cells.
+Only cells for which `counter ∈ counters` holds or the cell text matches `regex`
```

Comment:
Oops, you're right, in the code it's currently "and". If you think it's more usable (I think I do), we could change it to "or". However, that will incur more changes than turning `&&` into `||`, as e.g. `counter in 1:3 || ismatch(r"", s)` would match every cell, and not just cells 1 to 3. Probably change the defaults back to `nothing` and then change the condition to something like
    (counters==nothing && regex==nothing) ||
    (counters!=nothing && counter in counters) ||
    (regex!=nothing && ismatch(regex,s)) 
(At least, I don't see anything easier for the formulation of the condition)